validate/main.go

```diff
@@ -15,29 +15,29 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"strings"
 
 	config "github.com/coreos/ignition/config/v2_4_experimental"
 	"github.com/coreos/ignition/internal/version"
-
-	"github.com/spf13/cobra"
 )
 
 var (
 	flagVersion bool
-	rootCmd     = &cobra.Command{
-		Use:   "ignition-validate config.ign",
-		Short: "ignition-validate will validate Ignition configs",
-		Run:   runIgnValidate,
-	}
+	helpVersion bool
 )
 
 func main() {
-	rootCmd.Flags().BoolVar(&flagVersion, "version", false, "print the version of ignition-validate")
-	rootCmd.Execute()
+	rootCmd := flag.NewFlagSet("root", flag.ExitOnError)
+	rootCmd.BoolVar(&flagVersion, "version", false, "print the version of ignition-validate")
+	rootCmd.BoolVar(&helpVersion, "help", false, "print the version of ignition-validate")
```

Comment:
I don't think we actually need to define `--help`. See https://golang.org/pkg/flag/#pkg-variables. We can probably just override `Usage()` and rely on the default flag handling golang provides.